django/db/backends/postgresql_psycopg2/client.py

```diff
@@ -1,19 +1,63 @@
+from __future__ import print_function
+
+import os
 import subprocess
+import tempfile
 
 from django.db.backends.base.client import BaseDatabaseClient
 
 
+def _escape_pgpass(txt):
+    'Escape a non-terminal fragment of a postgresql .pgpass file'
+    return txt.replace('\\', '\\\\').replace(':', '\\:')
+
+
 class DatabaseClient(BaseDatabaseClient):
     executable_name = 'psql'
 
     def runshell(self):
         settings_dict = self.connection.settings_dict
         args = [self.executable_name]
-        if settings_dict['USER']:
-            args += ["-U", settings_dict['USER']]
-        if settings_dict['HOST']:
-            args.extend(["-h", settings_dict['HOST']])
-        if settings_dict['PORT']:
-            args.extend(["-p", str(settings_dict['PORT'])])
-        args += [settings_dict['NAME']]
+
+        host = settings_dict.get('HOST', '')
+        port = settings_dict.get('PORT', '')
+        name = settings_dict.get('NAME', '')
+        user = settings_dict.get('USER', '')
+        passwd = settings_dict.get('PASSWORD', '')
+
+        if user:
+            args += "-U", user
```

Comment:
can you explicitly wrap them in brackets: `args += ["-U", user]` please. That makes it clearer to understand the code.